src/index.ts

```diff
@@ -31,6 +32,7 @@ const COMMANDS = {
   herotag: 'herotag',
   pauseUnpauseEsdt: 'pause-unpause-esdt',
   freezeUnfreezeEsdt: 'freeze-unfreeze-esdt',
+  transferOwnershipESDT: 'transferOwnershipESDT',
```

Comment:
I would keep the same naming convention here for the value, so: `transferOwnershipESDT: transfer-ownership-esdt`